<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCreateUser</class>
 <widget class="QDialog" name="CCreateUser">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>169</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>454</width>
    <height>169</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Создание нового пользователя</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Login_Label">
       <property name="text">
        <string>Логин:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Password_Label">
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="RepeatPassword_Label">
       <property name="text">
        <string>Повторите пароль:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Login_Edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Rights_Label">
       <property name="text">
        <string>Мандатная группа:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="Rights_ComboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Password_Edit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="RepeatPassword_Edit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="ShowPassword_CheckBox">
     <property name="text">
      <string>Показать пароль</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="Confirm_Button">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
